packages/js/src/plugins/auctionHouseModule/operations/updateAuctionHouse.ts

```diff
@@ -48,19 +60,96 @@ export type UpdateAuctionHouseOperation = Operation<
  * @category Inputs
  */
 export type UpdateAuctionHouseInput = {
-  // Main Accounts.
+  /**
+   * The Auction House model.
+   * We need the full model in order to compare the current data with
+   * the provided data to update. For instance, if you only want to
+   * update the `feeWithdrawalDestination`, we need to send an instruction that updates
+   * the data whilst keeping all other properties the same.
+   *
+   * If you want more control over how this transaction is built,
+   * you may use the associated transaction builder instead using
+   * `metaplex.auctionHouse().builders().updateAuctionHouse({...})`.
```

Comment:
This is not quite right since the Transaction Builder also requires the full Auction House model I'm afraid. Let's just delete that paragraph for now.